zeppelin-web/src/app/spell/spell-base.js

```diff
@@ -31,9 +31,10 @@ export class SpellBase {
    * Consumes text and return `SpellResult`.
    *
    * @param paragraphText {string} which doesn't include magic
+   * @param config {Object}
    * @return {SpellResult}
    */
-  interpret(paragraphText) {
+  interpret(paragraphText, config) {
```

Comment:
@Leemoonsoo Thanks for review! 

- Config can change easily at anytime. 
- If we pass config to ctor, we need to create class everytime